tsconfig.build.json

```diff
@@ -3,9 +3,10 @@
   "compilerOptions": {
     "declaration": true,
     "declarationMap": true,
```

Comment:
If we're only emitting type definitions, would we still be emitting source maps for those type definitions?

Reply:
TypeScript still emits a source map. I'm not sure how to test if it's correct, but it seems alright?

```json
{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../src/index.ts"],"names":[],"mappings":"AAAA;;;;;GAKG;AACH,MAAM,CAAC,OAAO,UAAU,OAAO,CAAC,IAAI,EAAE,MAAM,GAAG,MAAM,CAEpD"}
```

Reply:
Hmm, I found this unexpected as well, as I don't know of a time I've needed to map a type to its source code. But I guess that makes sense.

Reply:
More context here: #182.

Reply:
Right — you should be able to test this by clicking on a symbol in an import and issuing the "Go to Source Definition" command and you should be directed to the JS file rather than the type definition file (follow the directions here: https://github.com/MetaMask/metamask-module-template/pull/182#issuecomment-1579817679).